src/generator.js

```diff
@@ -4,43 +4,125 @@ import { fileURLToPath } from 'url'
+}
+
 function generateHTML (sites) {
+  const totalCount = sites.length
+
   const htmlContent = `
     <!DOCTYPE html>
     <html lang="en">
     <head>
         <meta charset="UTF-8">
         <meta name="viewport" content="width=device-width, initial-scale=1.0">
         <meta name="description" content="Explore the Distributed Press ecosystem. Find sites leveraging decentralized publishing.">
+        <!-- icons -->
+        <link rel="icon" type="image/png" href="favicon-96x96.png" sizes="96x96">
+        <link rel="icon" type="image/png" href="favicon-32x32.png" sizes="32x32">
+        
         <title>Distributed Press Site Index</title>
         <link rel="stylesheet" href="styles.css">
     </head>
     <body>
-        <h1>Distributed Press Site Index</h1>
-        <p>Welcome to the Distributed Press Site Index. Below is a list of sites currently using Distributed Press.</p>
-        <ul>
-            ${sites
-              .map(
-                (site) => `
-                <li>
-                    <a href="${site.links.http?.link || '#'}" target="_blank">${site.domain}</a>
-                    <p><strong>Title:</strong> ${site.metadata.title}</p>
-                    <p><strong>Description:</strong> ${site.metadata.description}</p>
-                    <p><strong>Hyper Link:</strong> ${
-                      site.links.hyper?.enabled
-                        ? `<a href="${site.links.hyper.link}" target="_blank">${site.links.hyper.link}</a>`
+        <div class="container">
+          <!-- DP Logo -->
+          <div style="text-align: center; margin-bottom: 20px;">
+            <a href="//distributed.press" target="_blank">
+              <img
+                alt="Distributed Press"
+                src="//distributed.press/img/logos/logo-distributedpress-grey.png"
+                style="width: 200px; height: auto;"
+              />
+            </a>
+          </div>
+          <!-- Total Count -->
+          <p style="text-align: center; font-weight: bold;">Total Websites: ${totalCount}</p>
+          <!-- Welcome and Additional Links -->
+          <p>Welcome to the Distributed Press Site Index. Below is a list of sites currently using Distributed Press.</p>
+          <p style="margin-bottom: 20px; font-size: 12px;">
+            Didn't see your site here? Please make an 
+            <a href="https://github.com/hyphacoop/explore.distributed.press/issues" target="_blank">issue</a>. 
+            To view the native URLs, use P2P browsers like 
+            <a href="//agregore.mauve.moe/" target="_blank">Agregore</a> and 
+            <a href="//peersky.p2plabs.xyz/" target="_blank">Peersky</a>.
+          </p>
+          <!-- Table -->
+          <table>
+            <thead>
+              <tr>
+                <th>Domain</th>
+                <th>Title</th>
+                <th>Description</th>
+                <th>URLs</th>
+                <th>IPFS Hash</th>
```

Comment:
```suggestion
                <th>IPFS Public Key</th>
```


These are not hashes but public keys, hyper as well